js-guide.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Newbie Guide: JavaScript</title>
+  </head>
+  <body>
+    <h1>Newbie Guide: JavaScript</h1>
+    <h2>Brief history of language</h2>
+    <p>JavaScript is an interpreted programming language originally created in 1995 by Brendan Eich. It is known as the language of the web, but it can also be run outside of a web browser.</p>
+
+    <h2>Where is Javascript used? What can I program with it?</h2>
+    <ul>
+      <li>Browser / client-side (websites UI, single-page-application frameworks)</li>
+      <li>Server-side / non-browser runtimes (NodeJS)</li>
+      <li>Additional cross-platform applications (Electron, IoT, anything else)</li>
+    </ul>
+
+    <h2>Key concepts &amp; best practices w/ Javascript</h2>
+    <ul>
+      <li>DOM (Document Object Model)</li>
+      <li>Primitive types (explain these briefly?)</li>
+      <li>Prototypical Inheritance</li>
+      <li>Closure</li>
+      <li>Functions as first-class citizens</li>
+      <li>ES6/2015 and onward</li>
+      <li>Module Systems (brief explanation of what these are)</li>
+    </ul>
+
+    <h2>Learning Resources</h2>
+    <h3>Free E-Books</h3>
+    <ul>
+      <li><a href="http://eloquentjavascript.net/">Eloquent Javascript</a></li>
+      <li><a href="http://speakingjs.com/">Speaking Javascript</a></li>
+      <li><a href="https://addyosmani.com/resources/essentialjsdesignpatterns/book/">Javascript Design Patterns</a></li>
+    </ul>
```

Comment:
I think than EloquentJS and Rauschmeyer's Speaking JS are two of the best free books out there for beginners.  Not sure if Addy Osmani's book is, though.  I love that book and think that the idea of learning design patterns is essential - just not suitable for someone new to Javascript?  Maybe Eric Elliott's free e-book would be better here? http://chimera.labs.oreilly.com/books/1234000000262/index.html